src/ray/common/ray_config_def.h

```diff
@@ -491,9 +491,10 @@ RAY_CONFIG(int64_t, task_events_report_interval_ms, 1000)
 /// Setting the value to -1 allows for unlimited task events stored in GCS.
 RAY_CONFIG(int64_t, task_events_max_num_task_in_gcs, 100000)
 
-/// Max number of task events stored in the buffer on workers. Any additional events
-/// will be dropped. This is set to a large value to avoid worker side data loss.
-RAY_CONFIG(uint64_t, task_events_max_buffer_size, 100 * 1000)
+/// Max number of task events by type (e.g. status updates, profile events) stored on
+/// workers. Any additional events will be dropped. This is set to a large value to avoid
+/// worker side data loss.
+RAY_CONFIG(int64_t, task_events_max_num_events_by_kind_in_worker, 100 * 1000)
```

Comment:
Hmm this name is confusing to me. Can you explain to me what this exactly means?

Reply:
So there are two kinds:
1. profile events
2. status events.

This controls how many of these to be stored before dropping on the worker.